### DISCUSSION TITLE
POST_LOGIN_XX not working as expected

### DISCUSSION THREAD

**Discussion by Soylor** [ltsp/ltsp#804] [General] [answered]
Post login section not working.

I'm using a Jammy Ubuntu 22.04 image with this `LTSP.CONF`.
The POST_LOGIN commands are not executed

```
POST_LOGIN_FIREFOX="section_face"
[face]
 gsettings set org.gnome.shell favorite-apps "$(gsettings get org.gnome.shell favorite-apps | sed s/.$//), 'firefox.desktop']"
 gsettings set org.gnome.desktop.background picture-uri 'http://172.19.146.4/scripts/apariencia/uv-background.jpg'

```
I did test the commands in a  xterm after login and work if executed writing them.
Any help?

**Comment by maccadia** [comment, marked as answer]
You need to use a dconf profile : https://help.gnome.org/admin/system-admin-guide/stable/dconf-custom-defaults.html.en

gsettings change the configuration for the current user only ; not all users.

**Comment by alkisg** [comment]
LTSP currently has no "user" events. The LOGIN event runs from PAM as root; it's not suitable for running user apps like `gsettings`.

What MAccadia said is best; while for "post login" events there's also the usual /etc/xdg/autostart which isn't LTSP-specific.

**Comment by Soylor** [comment]
Thank you, Maccadia and alksig.
It works.
